Instructions_for_GSSOC.md

```diff
@@ -0,0 +1,13 @@
+# Instructions for Girlscript Summer of Code 2020
+- Introduce yourself on the Slack Channel.
+- Get in touch with Mentors(M) on Slack, if you are new to Git and Github and wish to contribute.
+- Review the Issues which are tagged with GSSOC. 
+- Before writing code do follow the CONTRIBUTING.md to follow the certain guidelines that must be adhered while contributing to the Project.
```

Comment:
Before contributing to the codebase, go through `CONTRIBUTING.md` to know the guidelines that must be followed/adhered.